README.md

```diff
@@ -94,15 +94,19 @@ pip install onnx
 ### Windows
 If you are building ONNX from source on Windows, it is recommended that you also build Protobuf locally as a static library. The version distributed with conda-forge is a DLL and this is a conflict as ONNX expects it to be a static library.
 
+Note that the instructions in this README assume you are using Visual Studio. It is recommended that you run all the commands from a shell started from "Developer Command Prompt for VS 2019" and keep the build system generator for cmake consistent.
+
 #### Build Protobuf and ONNX on Windows
 Step 1: Build Protobuf locally
 ```
 git clone https://github.com/protocolbuffers/protobuf.git
 cd protobuf
-git checkout 3.9.x
+git checkout 3.12.x
```

Comment:
I can build onnx on Windows with the latest protobuf so I updated it. Please comment if you have any concern about this. Thanks.

Reply:
Is this working with Windows Anaconda?

Reply:
cc @vinitra for checking with Windows Anaconda. Thanks!

Reply:
@wschin Even I just tested it. Even Protobuf 3.13 (shared libary) is working with Windows conda.

Reply:
please use 3.11.3 (as this is the one being used in the current release package)